modules/configuring-default-route-timeouts-globally.adoc

```diff
@@ -10,17 +10,35 @@ By configuring the route timeouts globally, you can ensure consistent timeout se
 
 You can configure the route timeouts globally by updating the `ROUTE_HAPROXY_TIMEOUT` environment value in your `serverless-operator` subscription and updating the `max-revision-timeout-seconds` field in your `KnativeServing` custom resource (CR). This applies the timeout changes across all Knative services, and you can deploy services with specific timeouts up to the maximum value set.
 
+`ROUTE_HAPROXY_TIMEOUT` is an environment variable managed by the Serverless Operator and by default is set to `600`.
+
 .Procedure
 
-. Set the value of `ROUTE_HAPROXY_TIMEOUT` in your subscription to your required timeout in seconds by running the following command:
+. Set the value of `ROUTE_HAPROXY_TIMEOUT` in your subscription to your required timeout in seconds by running the following command. Note that this causes pods to be redeployed in the `openshift-serverless` namespace.
```

Comment:
Another change for later. As you have it, one could read this to mean that all pods in all namespaces are redeployed to the `openshift-serverless` namespace. 
```suggestion
. Set the value of `ROUTE_HAPROXY_TIMEOUT` in your subscription to your required timeout in seconds by running the following command. Note that this causes pods in the `openshift-serverless` namespace  to be redeployed.
```